libs/data-mapper-v2/src/components/schema/SchemaPanel.tsx

```diff
@@ -167,9 +167,9 @@ export const SchemaPanel = ({ onSubmitSchemaFileSelection, schemaType }: ConfigP
         return;
       }
 
-      if (uploadType === UploadSchemaTypes.SelectFrom && selectedSchema) {
+      if (fileSelectorOptions === 'select-existing' && selectedSchema) {
```

Comment:
Discussed moving this to the child component- Krrish will update this in his next PR!